host/common/msg.c

```diff
@@ -423,6 +459,17 @@ void bl_msg_yaml_print(FILE *file, const union bl_msg_data *msg)
 		}
 		break;
 
+	case BL_MSG_VERSION_REQ:
+		break;
+
+	case BL_MSG_VERSION:
+		fprintf(file, "    Revision: %"PRIu8"\n", msg->version.revision);
+		fprintf(file, "    Commit Sha: ");
+		for (unsigned i = 0; i < COMMIT_SHA_LENGTH; i++) {
+			fprintf(file, "%"PRIx32, msg->version.commit_sha[i]);
```

Comment:
Maybe `"%8.8"PRIx32`, to ensure leading zeros aren't skipped at the start of a block of 8.

Reply:
I managed to get this working with `"%08"PRIx32`, what would %8.8 do compared to this?

Reply:
`"%08"PRIx32` will work just as well. I usually do `"%8.8"PRIx32`, setting the field width and precision, for decimals, so the reader doesn't have to know how they're each defined.

Reply:
Okay, this should be sorted now.